### PR TITLE
Fixes v2

### PR DESCRIPTION
**Pull request by DetroitWhiskey136**
**Please describe the changes this PR makes and why it should be merged:**
Bug Fixes and other stuff.

**Status**
some of these have breaking changes. store file info in particular will need to be updated by the end user.

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/store_file_info_MOD.js [hunk 1]
```diff
@@ -70,7 +65,8 @@ Variable name:<br>
 
   async action(cache) {
     const data = cache.actions[cache.index];
-    const storage = parseInt(data.storage, 10);
+    const storage = this.eval(data.storage, 10);
+    console.log(storage);
```
Comment (almeidx) [inline actions/store_file_info_MOD.js:69]: 🤔

**Review by almeidx — CHANGES_REQUESTED** [review]
- Loop queue needs to be removed from the docs file
- The dashboard extension needs to be removed from the docs file
- The changes on the Store File Info are breaking, not ideal

**Comment by master3395** [comment]
set_time_restriction_MOD.js
Still fails on buttons, it does not seem to store any name.
image

File: actions/store_file_info_MOD.js (hunk 1 above)
Comment (DetroitWhiskey136) [reply actions/store_file_info_MOD.js:69]: not sure when that happened hmmmm

Comment (DetroitWhiskey136) [reply actions/store_file_info_MOD.js:69]: as for the `The changes on the Store File Info are breaking, not ideal` not sure how to do that without making it a breaking change as storage is always a number... I guess we could just not do the check but that seems wrong.

**Comment by DetroitWhiskey136** [comment]
> set_time_restriction_MOD.js Still fails on buttons, it does not seem to store any name. image

I don't believe that was the focus of this bug fix

File: actions/check_if_member_MOD.js [hunk 2]
```diff
@@ -136,6 +136,8 @@ module.exports = {
     const { Files } = this.getDBM();
     const { msg } = cache;
 
+    if (!member) return console.error('You need to provide a member of some sort.');
```
Comment (almeidx) [inline actions/check_if_member_MOD.js:139]: I believe we should still continue the action sequence. Perhaps a more informative error message would help too (at least show the action name)?

**Review by almeidx — APPROVED** [review]

**Pull request merged**